MekHQ/src/mekhq/campaign/universe/PlanetarySystem.java

```diff
@@ -61,6 +60,77 @@
 @JsonIgnoreProperties(ignoreUnknown=true)
 @JsonDeserialize(converter= PlanetarySystem.PlanetarySystemPostLoader.class)
 public class PlanetarySystem {
+
+    
+    // --- Sophistication Rating Enum ---
+    public enum PlanetarySophistication {
+        ADVANCED(0, "Advanced"),
+        A(1, "A"),
+        B(2, "B"),
+        C(3, "C"),
+        D(4, "D"),
+        F(5, "F"),
+        REGRESSED(6, "Regressed");
+
+        private final int index;
+        private final String name;
+        private static final Map<Integer, PlanetarySophistication> INDEX_LOOKUP = new HashMap<>();
+        private static final Map<String, PlanetarySophistication> NAME_LOOKUP = new HashMap<>();
+        static {
+            for (PlanetarySophistication tr : values()) {
+                INDEX_LOOKUP.put(tr.index, tr);
+                NAME_LOOKUP.put(tr.name, tr);
+            }
+        }
+        PlanetarySophistication(int idx, String name) { this.index = idx; this.name = name; }
+        public int getIndex() { return index; }
+        public String getName() { return name; }
+        public static PlanetarySophistication fromIndex(int idx) {
+            PlanetarySophistication tr = INDEX_LOOKUP.get(idx);
+            if (tr == null) throw new IllegalArgumentException("Invalid PlanetarySophistication index: " + idx);
+            return tr;
+        }
+        public static PlanetarySophistication fromName(String name) {
+            PlanetarySophistication tr = NAME_LOOKUP.get(name);
+            if (tr == null) throw new IllegalArgumentException("Invalid PlanetarySophistication name: " + name);
+            return tr;
+        }
+    }
+
+    // --- Planetary Rating Enum ---
+    public enum PlanetaryRating {
+        A(0, "A"),
+        B(1, "B"),
+        C(2, "C"),
+        D(3, "D"),
+        F(4, "F");
+
+        private final int index;
+        private final String name;
```

Comment:
I'd like to see these moved to their own files under a PlanatarySystem package (or something similar).